### ISSUE TITLE
Relative_action

### ISSUE THREAD

**Issue by chenjiayi-zxx** [nickgkan/3d_diffuser_actor#25]
Hello, I am currently exploring your fascinating project and I have noticed that the relative_action parameter is specifically utilized with CALVIN. I am curious to understand the rationale behind this specific usage.
﻿
Could you kindly share the insights or analyses that led to the decision to use relative_action only in the context of CALVIN? 
﻿
Thank you very much for your time and for the impressive work on this project.

**Comment by nickgkan** [comment]
Hi @chenjiayi-zxx, thanks for you interest in our project!

In some earlier experiments we found using a relative coordinate frame not to be an important factor on RLBench. However, we did find it important on CALVIN, as also suggested by related literature such as [HULC](https://arxiv.org/pdf/2204.06252.pdf). We hypothesize that this is due to the nature of instructions of CALVIN, e.g. "mover block to the right", which suggest relative actions. Instead, on RLBench there is usually a set of solutions defined visually, e.g. reaching the handle of a door.

We haven't retried relative actions recently on RLBench, so it's possible that if you try you'll notice some performance difference.
